virridy/verification_workflow/raw_data_plotter.R

```diff
@@ -0,0 +1,158 @@
+    #data <- readRDS("~/Documents/fork_yeah/poudre_sonde_network/data/virridy_verification/all_data_flagged_complete.RDS")
+    # Filter the data based on user input
+
+    #grab the site and parameter combos the user selected
+
+    sites_selected <- c(input$site_oi, input$selected_site)
+
+    site_param_combos <- crossing(site_select = sites_selected, param = input$selected_param) %>%
+      mutate(combo = paste(site_select, param, sep = "-")) %>%
+      pull(combo)
+
+    #grab the data for the site and parameter combos the user selected
+    all_select_data <- site_param_combos %>%
+      keep(~ .x %in% names(data)) %>%
+      map_dfr(~ data[[.x]])
+    #grab start and end dates
+    start_date <- as.POSIXct(input$date_range[1])%>% force_tz("MST")
+    end_date <- as.POSIXct(input$date_range[2])%>% force_tz("MST")
+
+    # # Do any transformations...
+    # if(input$transformation == "None"){
+    #   #trim to just the dates selected
+    trim_select_data <- all_select_data %>%
+      filter(DT_round >= start_date & DT_round <= end_date)%>%
+      #convert depth to ft (easier to see changes than meters)
+      mutate(mean = case_when(parameter == "Depth"~ mean*3.28084,
+                              TRUE ~ mean))%>%
+      #remove rows where "missing data" is included in the flag string but still include rows with NA
+      filter(grepl("missing data", flag) == FALSE | is.na(flag))
+
+
+    #Create the baseplot that will be customized below
+    base_plot <- ggplot() +
+      geom_line(data = trim_select_data %>% filter(site != input$site_oi),
+                aes(x = DT_round, y = mean, color = site)) +
+      facet_wrap(~parameter, scales = "free_y", ncol = as.integer(input$col_number)) +
+      labs(x = "Date", y = "Value") +
+      theme_bw()
+
+    output$data_plot <- renderPlotly({
+
+    if(input$transformation == "None"){
+
+        # Base plot
+        if(input$flags_TF == "yes"){
+          p <- base_plot +
+            geom_point(data = trim_select_data %>% filter(site == input$site_oi),
+                       aes(x = DT_round, y = mean, colour = flag)) +
+            labs(colour = "Flags on data", color = "Additional Sites")
+
+
+        } else {
+          p <- base_plot +
+            geom_point(data = trim_select_data %>% filter(site == input$site_oi),
+                       aes(x = DT_round, y = mean, colour = site)) +
+            labs(colour = "Site OI", color = "Additional Sites")
+
+        }
+
+    }
+### Log transformed plot ####
+    if(input$transformation == "log10()"){
+
+        # Plot data and apply log scale to selected parameters
+        if(input$flags_TF == "yes"){
+
+          # Flags + log 10
+          p <- base_plot +
+            geom_point(data = trim_select_data %>% filter(site == input$site_oi),
+                       aes(x = DT_round, y = mean, colour = flag)) +
+
+            labs(colour = "Flags on data", color = "Additional Sites")+
+            scale_y_continuous(trans = "log",
```

Comment:
```suggestion
            scale_y_continuous(trans = "log10",
```

Reply:
This is a screw up I did! Setting to base 10 log. Shouldn't impact things too much visually at all.